src/main/java/pro/sky/Course2SpringShop/model/Product.java

```diff
@@ -0,0 +1,31 @@
+package pro.sky.Course2SpringShop.model;
+
+import org.springframework.stereotype.Component;
+
+import java.util.Objects;
+@Component
+public class Product {
+    private String productName = "";
```

Comment:
```suggestion
    private String productName;
```
зачем пустая строка?